AmazonPublisherServicesAdapter/build.gradle.kts

```diff
@@ -42,7 +42,7 @@ android {
         minSdk = 21
         targetSdk = 34
         // If you touch the following line, don't forget to update scripts/get_rc_version.zsh
-        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "5.9.9.5.0"
+        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "5.9.9.5.1"
```

Comment:
we haven't published 5.9.9.5.0 so we can keep it at 5.9.9.5.0

Reply:
Good call